utils/emacs/README.md

```diff
@@ -0,0 +1,20 @@
+# Carbon-mode
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+`carbon-mode` provides Carbon syntax highlighting for Emacs.
+
+## Installation
+
+Currently only manual installation is supported.
+
+### Manual installation
+
+```elisp
+(add-to-list 'load-path "/path/to/carbon-mode/")
+(autoload carbon-mode "carbon-mode" nil t)
```

Comment:
Maybe add the auto-mode-alist elisp here? That way the mode is loaded automatically the first time a `.carbon` file is opened. Alternatively, just `(require 'carbon-mode)` as the autoload stuff is just an optimization.

Reply:
Yeah. I should just use `require` here (ironically `require` is what I use in my config noew)